sql/sqlwire/sql_api.proto

```diff
@@ -93,6 +93,9 @@ message SQLRequest {
 
 message SQLResponse {
   optional SQLResponseHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
+  // The names of the columns returned in the result set.
+  // The number of columns must equal the number of Datum in each Result.
```

Comment:
Columns are ordered by how they are requested in the SQL statement. And if ordering is unspecified there (e.g. `SELECT * FROM foo`) then they are ordered by how they were defined in the table. That is, `SELECT a, b FROM foo` will return the columns in the order `a`, `b`. And `SELECT b, a FROM foo` will return the columns in the order `b`, `a`. You can also do fun stuff like retrieve the same column multiple times: `SELECT a, a, a FROM foo`.